### ISSUE TITLE
[BUG][MARKDOWN] Extra (back)slashes added to links from Api to Model

### ISSUE THREAD

**Issue by zeljkot** [OpenAPITools/openapi-generator#6095]
##### Description

Links from Apis to Models directories have additional (back)slash that breaks the links. That happens for both parameters and return types. For example:

```markdown
### Parameters

Name | Type | Description  | Notes
------------- | ------------- | ------------- | -------------
 **ids** | [**List**](../\Models/Integer.md)|  | [default to null]
```

```Markdown
### Return type

[**List**](../\Models/string.md)
```

Same happen on Linux:  `[**List**](..//Models/string.md)`

##### openapi-generator version

4.3.0

##### OpenAPI declaration file content or url

[bug.zip](https://github.com/OpenAPITools/openapi-generator/files/4550218/bug.zip)

##### Command line used for generation

```xml
<groupId>org.openapitools</groupId>
<artifactId>openapi-generator-maven-plugin</artifactId>
<version>4.3.0</version>
<executions>
  <execution>
    <id>markdown</id>
```

`mvn clean compile`

##### Steps to reproduce

Create OpenAPI yaml with components.

##### Suggest a fix

**Comment by ybelenko** [comment]
I confirm this bug. I would fix it myself, but don't remember should we delete `File.separator` usage or not. I think @jimschubert as author of markdown generator can help. He more skilled in Java and with path resolving than me.

**Comment by jimschubert** [comment]
The fix contributed in #6834 is now in master
